src/main/resources/react/src/components/pages/homepage/slider/Sliders.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import "./slider.scss";
+import toySlider from "../../../../img/slider/toy.jpg";
+import clothesSlider from "../../../../img/slider/clothes.jpg";
+import childSlider from "../../../../img/slider/child.jpg";
+import furnitureSlider from "../../../../img/slider/furniture.jpg";
+import shoesSlider from "../../../../img/slider/shoes.jpg";
+import strollersSlider from "../../../../img/slider/strollers.jpg";
+import booksSlider from "../../../../img/slider/books.jpg";
```

Comment:
Is it possible to use generic import something like that ``import blabla from "../../../../img/slider/**";`` or you have to import all images one by one?

Reply:
to fixed